services/policies/pkg/engine/opa/engine.go

```diff
@@ -17,36 +19,49 @@ type OPA struct {
 	printHook print.Hook
 	policies  []string
 	timeout   time.Duration
+	options   []func(r *rego.Rego)
 }
 
 // NewOPA returns a ready to use opa engine.
 func NewOPA(timeout time.Duration, logger log.Logger, conf config.Engine) (OPA, error) {
+	var mtReader io.Reader = nil
```

Comment:
```suggestion
	var mtReader io.Reader
```
This already defaults to nil!?

Reply:
yes empty interface, THX